### ISSUE TITLE
Mac ARM version?

### ISSUE THREAD

**Issue by nesdnuma** [CircuitMusicLabs/petra#34]
Hello,

I usually teach granular synthesis with both MuBu and Petra to my students.

Some students recently purchased M1 Mac computers and cannot use Petra. Do you plan to release some version compiled for M1?

**Comment by CircuitMusicLabs** [comment]
Hi nesdnuma
Thanks for you message. Unfortunately, I have bad news for you. You woke me up in deep hybernation.
To be absolutely honest, my life took a complete turn in the last few years and I have not dabbled with audio development for a very long time. Also, I currently lack the Mac hardware to even think about rebuilding the current codebase. So this is why I have to tell you that I won't be able to - and to be frank I am not willing to - get things back up running again.
Your message spurred me into action, however. I have since updated the readme to reflect the current state of the project and that I have discontinued support. I have also informed Cycling'74 about this.
On this note, I would also like to tell you that the project is up for grabs, so if you - or anyone you know - are interested in taking over the codebase, be free to do so. Maybe one of your students would be up for a challenge?
Thanks again for your message. I wish you and your students all the best.
Matthias
